mk/px4_targets.mk

```diff
@@ -125,6 +125,11 @@ px4-cleandep: clean
 	$(v) find $(UAVCAN_DIRECTORY) -type f -name '*.d' | xargs rm -f
 	$(v) find $(SKETCHBOOK)/$(SKETCH) -type f -name '*.d' | xargs rm -f
 
+px4-v2-upload-solo: px4-v2
+	scp $(SKETCH)-v2.px4 root@10.1.1.10:/tmp/
```

Comment:
I see what you mean. I don't like the hard coded approach though as it becomes necessary to keep track of changes to `SKETCH`. Maybe we could get to a solution that satisfies both of us? :-)